### DISCUSSION TITLE
Does Maui splash screen support full background image?

### DISCUSSION THREAD

**Discussion by gabsamples6** [dotnet/maui#19177] [General]
Hi
specially android I cannot seem to be able to show a background image in the splash screen .Is this supported?
What I mean is a custom background image that takes the full screen supported?>

Reading the google guidelines https://developer.android.com/develop/ui/views/launch/splash-screen 
I am not sure and if it is supported how do you do in Maui?

Does iOS support it?

thanks

**Comment by Xyncgas** [comment]
I guess a splash screen svg was included in your MAUI app, if you replace it with another svg as long they are the correct dimension, it should always auto scale
